de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/fresh_PriceList_Details_Template_Report.sql

```diff
@@ -11,37 +11,55 @@ CREATE OR REPLACE FUNCTION report.fresh_pricelist_details_template_report(IN p_c
-       plc.productname                                                           as productname,
-       plc.attributes                                                            as attributes,
-       plc.itemproductname                                                       as itemproductname,
+
+SELECT plc.value                                                                                                              AS prodvalue,
+       plc.customerproductnumber                                                                                              as customerproductnumber,
+       plc.productcategory                                                                                                    as productcategory,
+       plc.productname                                                                                                        as productname,
+       plc.attributes                                                                                                         as attributes,
+       case
+           when coalesce(hupiv.description, hupip.name) is null then plc.uomsymbol
+           else
+               coalesce(hupiv.description, hupip.name) end                                                                    as itemproductname,
        -- if packing instructions exist, we will display the qty one as it refers to one TU (Transportation Unit).
-       CASE WHEN plc.itemproductname IS NOT NULL THEN 1 ELSE plc.qtycuspertu END AS qty,
-       plc.uomsymbol                                                             as uomsymbol,
-       plc.pricestd                                                              as pricestd,
-       plc.M_ProductPrice_ID                                                     as m_productprice_id,
-       p_c_bpartner_id                                                           as c_bpartner_id,
-       plc.M_HU_PI_Item_Product_ID                                               as m_hu_pi_item_product_id,
-       plc.UOM_X12DE355                                                          as uom_x12de355,
-       p_c_bpartner_location_id                                                  as c_bpartner_location_id,
-       plc.qtycuspertu                                                           as qtycuspertu,
-       plc.M_Product_ID                                                          as m_product_id
+--        CASE WHEN plc.itemproductname IS NOT NULL THEN 1 ELSE plc.qtycuspertu END AS qty,
+       NULL::numeric                                                                                                          as qty,
+       plc.uomsymbol                                                                                                          as uomsymbol,
+       cast(plc.pricestd as decimal(19, 2))::text                                                                             as pricestd,
```

Comment:
not sure i like `as decimal(19, 2)` : since this is a price, shouldn't we round in jasper the number to the currency's standard rounding precision instead of a hardcoded value? what happens if the customer changes that precision at some point?
![image](https://user-images.githubusercontent.com/4482210/77397753-d8f6a000-6dae-11ea-9b9a-565685e37062.png)